packages/augur-ui/src/modules/account/components/account-view.tsx

```diff
@@ -113,6 +115,35 @@ export default class AccountView extends React.Component<
                 <Transactions />
                 <TermsAndConditions />
               </div>
+            ) : (
+              <div
+                className={classNames(Styles.AccountViewWrap)}
+              >
+                <div>
+                  My Account
+                </div>
+                <div
+                  className={classNames(Styles.AccountView, {
+                    [Styles.HideNotifications]: extendActiveMarkets,
+                    [Styles.HideTransactions]: extendWatchlist,
+                    [Styles.HideActiveMarkets]: extendNotifications,
+                  })}
+                >
+                  <div>
+                    <Notifications />
+                    <OpenMarkets />
+                  </div>
+                  <div>
+                    <Overview hideHeader={false} />
+                    <AugurStatus hideHeader={false} />
+                  </div>
+                  <div>
+                    <Transactions />
+                    <Favorites />
+                  </div>
+                </div>
+                <TermsAndConditions />
+              </div>
```

Comment:
I'd simplify this to just a div with `AccountView` and everything inside of it with no added divs for row separation. Use grid to place the pieces where they belong.

Reply:
Updated. Some UIs have not been updated yet. I will work on it.